spec/support/test_pals.rb

```diff
@@ -0,0 +1,108 @@
+# This class defines a set of users, schools, homerooms and students
+# that can be used for testing authorization rules.
+#
+# These can be re-used for any other test code, but changes here will impact
+# many tests, so the intention is that these should not change frequently.
+# If new attributes are added to models, update the factories instead.
+class TestPals
+  def self.create!
+    pals = TestPals.new
+    pals.create!
+    pals
+  end
+
+  attr_reader :uri
+  attr_reader :healey
+  attr_reader :healey_kindergarten_student
+  attr_reader :healey_teacher
+  attr_reader :healey_ell_teacher
+  attr_reader :healey_sped_teacher
+  attr_reader :healey_kindergarten_homeroom
+  attr_reader :shs
+  attr_reader :shs_freshman_mari
+  attr_reader :shs_jodi
+  attr_reader :shs_bill_nye
+  attr_reader :shs_ninth_grade_counselor
+  attr_reader :shs_jodi_homeroom
+  attr_reader :shs_biology_course
+  attr_reader :shs_tuesday_biology_section
+  attr_reader :shs_thursday_biology_section
+
+  def create!
+    School.seed_somerville_schools
+
+    # Uri works in the central office, and is the admin for the entire
```

Comment:
Sort of like little stories.  There's lots of details here, so thought that would help us make sense of this, and also let us talk about it like real rules.

Reply:
@kevinrobinson Great! I love these.